examples/integrated_gradients_transformers.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #x_test_sample = ["I love you, I like you",
</pre><p>remove lines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/430/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>